gflownet/gflownet.py

```diff
@@ -604,43 +536,55 @@ def trajectorybalance_loss(self, it, batch, loginf=1000):
         flow_loss : float
             Loss of the intermediate nodes only
         """
-        loginf = self._tfloat([loginf])
+        loginf = tfloat([loginf], device=self.device, float=self.float)
+        # Convert lists in the batch into tensors
+        batch.process_batch()
+
         # Unpack batch
-        (
-            states,
-            actions,
-            parents,
-            parents_a,
-            done,
-            traj_id_parents,
-            state_id,
-            masks_sf,
-            masks_b,
-        ) = zip(*batch)
+        # (
+        #     states,
+        #     actions,
+        #     parents,
+        #     parents_a,
+        #     done,
+        #     traj_id_parents,
+        #     state_id,
+        #     masks_sf,
+        #     masks_b,
+        # ) = zip(*batch)
```

Comment:
Could we remove this commented block?

Reply:
Of course! Sorry, I forgot about cleaning them up